tinyasm/tinyasm.cpp

```diff
@@ -63,31 +63,31 @@ class BlockJacobi {
                 fwork = vector<PetscScalar>(biggestBlock, 0.);
                 localmats_aij = NULL;
                 dofis = vector<IS>(numBlocks);
-                PetscMalloc1(numBlocks, &localmats);
+                PetscCallVoid(PetscMalloc1(numBlocks, &localmats));
```

Comment:
Must be `PetscCallVoid` because constructors/destructors cannot return anything.